CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+## How to contribute to `gleam`
+
+Thank you for taking the time to contribute to `gleam`!
+
+### Join
+
+The `gleam` community follows the [astropy code of conduct](https://www.astropy.org/code_of_conduct.html).
```

Comment:
Astropy

Reply:
Good point! I changed it to use the full name of their article: Astropy Community Code of Conduct.